chore: update licensing and docker token command (#6018)
influxdata/docs-v2: content/shared/v3-enterprise-get-started/_index.md

#### Licensing

> On first start of a new instance, you'll be prompted to select a license type.

Reviewer: ```suggestion
When first starting a new instance, InfluxDB prompts you to select a license type.
```

For more information, see how to [Manage admin tokens](/influxdb3/version/admin/tokens/admin/).

> [!Note]
> ##### Create a token interactively in Docker
> 
> If you're running Docker, you'll need to run commands in the container's shell. You can do this by

> > running `docker exec -it CONTAINER_NAME influxdb3 create token --admin` in a new tab.

Reviewer: ~~~suggestion
> If using Docker, open a new terminal session and run the following command to generate an
> {{% token-link "admin" "admin/" %}} in your container:
>
> <!-- pytest.mark.skip -->
> ```bash
> docker exec -it CONTAINER_NAME influxdb3 create token --admin
> ```
~~~

Author: I've updated this section to how it's done in the Processing Engine docs. Let me know if you want to do it differently there too.
